Apps/APN-UIKit/APN UIKit/Model/Settings.swift

```diff
@@ -2,9 +2,10 @@ import Foundation
 
 struct Settings {
     var deviceToken: String
-    var trackUrl: String
+    var cdnHost: String
+    var apiHost: String
     var siteId: String
-    var apiKey: String
+    var cdpWriteKey: String
```

Comment:
why not just `writeKey`? 😅

Reply:
To make it specific that this writeKey is for CDP. 😅 Does `cdpWriteKey` sound odd?

Reply:
[Updated](https://github.com/customerio/customerio-ios/pull/510/commits/6a83a73f72c73fdf55f860add4f376c4ab50c559)